app/routes/reject-application-claim.js

```diff
@@ -1,5 +1,6 @@
 const Joi = require('joi')
 const { processApplicationClaim } = require('../api/applications')
+const getUser = require('../auth/get-user')

```

Comment:
Can we put this in session once successfully logged in?

Reply:
@govTechSatish I guess we could but I'm not sure how you do this tbh. Can we get this merged so this isn't blocking testing and then maybe raise a task to add user to session?

Reply:
Sure. Create small ticker to cover this backlog. Approving for now.